### ISSUE TITLE
SDK openFile does not work with '+' in the file path

### ISSUE THREAD

**Issue by davidtlee** [stackblitz/core#2181]
### **Description of Bug**

The Stackblitz SDK does not successfully open the file passed in through `openFile` when there is a '+' in the path. For example, take the Stackblitz project https://stackblitz.com/edit/datetime-container-data which has files like `src/app/main/page/+state/page.model.ts`. 

### **Steps to Reproduce**

1. Use the Stackblitz SDK to open the project:
```
        sdk.embedProjectId('embed', 'datetime-container-data', {
          openFile: 'src/app/main/page/+state/page.model.ts',
        });
```

2. The file does not open (it just opens the README)

### **Expected Behavior**
We expect it to open the desired file (it works when trying other file paths without the '+')

**Comment by fvsch** [comment]
Hi @davidtlee,

I made a reproduction here:
https://stackblitz.com/edit/repro-stackblitz-core-2181

It looks like this was an issue with `@stackblitz/sdk` up to version 1.6.0, but was fixed in version 1.7.0.

Can you check your SDK version, and update to 1.7 or 1.8 maybe?

**Comment by fvsch** [comment]
Closing this issue because it appears to be fixed in versions 1.7+ of the SDK.

**Comment by davidtlee** [comment]
@fvsch sorry I took so long to respond. You're right - I had a very old version! Thank you :)
